.travis.yml

```diff
@@ -611,3 +612,5 @@ before_script:
   - gem install inspec rubocop
 script:
   - tox
+after_script:
+  - ./bin/rm-hetzner-cloud-token $HCLOUD_TOKEN
```

Comment:
Looking at this, we might want to use `commands_pre` and `commands_post` in `tox.ini` for this to keep the only source for truth.

Reply:
Right, that seems like a much better approach ... can hopefully avoid the scripts, maybe ...